Automattic/wp-calypso
# "Recurring Payments" -> "Payments"

This PR:
- Renames "Recurring Payments" to "Payments"
<img width="524" alt="Zrzut ekranu 2020-06-8 o 15 29 32" src="https://user-images.githubusercontent.com/3775068/84036238-402ef400-a99d-11ea-8878-f6d03eeb4aa6.png">
<img width="538" alt="Screen Shot 2020-06-17 at 3 49 55 PM" src="https://user-images.githubusercontent.com/1270189/84958429-35b5de00-b0b2-11ea-9976-76ff7d248fd5.png">
<img width="1037" alt="Zrzut ekranu 2020-06-8 o 15 29 53" src="https://user-images.githubusercontent.com/3775068/84036247-42914e00-a99d-11ea-816b-471e647a7fe5.png">
<img width="1038" alt="Zrzut ekranu 2020-06-8 o 15 30 06" src="https://user-images.githubusercontent.com/3775068/84036255-44f3a800-a99d-11ea-8f2c-8120d36009c1.png">
<img width="948" alt="Screen Shot 2020-06-17 at 3 52 37 PM" src="https://user-images.githubusercontent.com/1270189/84958653-aceb7200-b0b2-11ea-8ac0-a1201c50e956.png">
<img width="957" alt="Screen Shot 2020-06-17 at 3 52 44 PM" src="https://user-images.githubusercontent.com/1270189/84958655-ae1c9f00-b0b2-11ea-9172-6b1291b36afd.png">

### Notes
* I'll handle the one-time payment option in a seperate PR
* Modifying the copy in the following card, will be handled by the Simple Payments rename PR

<img width="542" alt="Screen Shot 2020-06-17 at 3 53 35 PM" src="https://user-images.githubusercontent.com/1270189/84958741-df956a80-b0b2-11ea-8ac7-07db978e3765.png">


## Testing instructions
- Sandbox, Set-up recurring payments
- Check out this PR
- Verify no copy changes in non-production environment, or add `?flags=-earn/rename-payment-blocks` and refresh
- Verify that copy changes make sense in development
